README.md

```diff
@@ -266,8 +281,10 @@ graph TB;
   R-. a,b,c prefer L .->L
 ```
 
-If a member `b` in `R` adds a new member `e` to `R`, then `b` MUST add `e`
+If a member `b` in `R` adds a new member `e` to `R`, then when `b` MUST add `e`
 to `L` as soon as `b` detects the existence of `L` (figure 3).
+> WARNING - this is technically 4.5 now - a subset membership T_T
```

Comment:
Nope this isn't subset membership. "commons" is not "intersection". `e` is basically invisible to this calculation, and it's still "same common membership".